src/bust.py

```diff
@@ -455,5 +459,6 @@ async def create_controller(
             )
             await discord_utils.try_set_pin(form_message, True)
 
+    await interaction.delete_original_message()
```

Comment:
A part of why I felt it was better to put them inside the BustController is that error messages are send in the bust controller, and it seemed cleaner for a function to return a bust controller where the interaction is definitely handled internally (either by error, by deleting command, by responding ok, etc), instead of one where whether or not it's been handled relies on whether or not the function succeeds.

The most proper way to really decouple all this IMO would be to return several different error codes and handle all those messages outside of the BustController, but that might be overengineering for our use case